message_definitions/v1.0/planckAero.xml

```diff
@@ -185,5 +203,17 @@
 	        <field type="uint8_t" name="batt_status" enum="PLANCK_DECK_BATTERY_STATUS_TYPE">Status of the DECK battery. See PLANCK_DECK_BATTERY_STATUS_TYPE enum</field>
 	        <field type="uint8_t" name="fourhundredvolt_status">Status of the DECK 400v output. 0: Off 1: On</field>
         </message>
+        <message id="9012" name="PLANCK_DECK_TETHER_STATUS">
+           <description>Status of the TETHER system</description>
+           <field type="float" name="CABLE_OUT">Number of feet of cable spooled out</field>
+           <field type="float" name="CABLE_VELOCITY">Current cable speed in ft/s</field>
+           <field type="uint8_t" name="CABLE_TENSION">Current cable tension as percentag of max</field>
+           <field type="uint8_t" name="HV_POWER">Status of high voltage subsystem (see PLANCK_DECK_TETHER_HV_STATUS_TYPE)</field>
```

Comment:
You can specify how this is described by adding an `enum="PLANCK_DECK_TETHER_HV_STATUS_TYPE"`